anycable-rails.gemspec

```diff
@@ -25,6 +25,8 @@ Gem::Specification.new do |spec|
   spec.files         = `git ls-files README.md MIT-LICENSE CHANGELOG.md lib`.split
   spec.require_paths = ["lib"]
 
+  spec.required_ruby_version = ">= 2.3"
```

Comment:
`Ruby >= 2.4` is mentioned in the docs

Reply:
Yeah, `anycable` requires 2.4+, so should we do here

https://github.com/anycable/anycable/blob/b1d0fe0cd125fe3ea02dcd04059d6a30ae63a918/anycable.gemspec#L29